### PR TITLE
chore: code cleanup

### PR DESCRIPTION
**Pull request by miquelbeltran**
### Description :memo:
- **Purpose**: Multiple small code cleanups.
- **Approach**: Different commits for small stuff that came in the late review of https://github.com/MindscapeHQ/raygun4node/pull/210

**Type of change**

- [x] Chore

**Updates**

- Fixes: https://github.com/MindscapeHQ/raygun4node/pull/210#discussion_r1604284723
- Fixes: https://github.com/MindscapeHQ/raygun4node/pull/210#discussion_r1604285893
- Fixes: https://github.com/MindscapeHQ/raygun4node/pull/210#discussion_r1604293833
- Fixes: https://github.com/MindscapeHQ/raygun4node/pull/210#discussion_r1604297348
- Fixes: https://github.com/MindscapeHQ/raygun4node/pull/222#discussion_r1604243868

### Test plan :test_tube:

- Run example. CI jobs should pass as well. Changes are mostly cosmetic.

### Author to check :eyeglasses:
- [x] Project and all contained modules builds successfully
- [x] Self-/dev-tested 
- [x] Unit/UI/Automation/Integration tests provided where applicable
- [x] Code is written to standards
- [x] Appropriate documentation written (code comments, internal docs)

### Reviewer to check :heavy_check_mark:
- [ ] Project and all contained modules builds successfully
- [ ] Change has been dev-/reviewer-tested, where possible
- [x] Unit/UI/Automation/Integration tests provided where applicable
- [x] Code is written to standards
- [x] Appropriate documentation written (code comments, internal docs)

### CODE REVIEW

**Review by TheRealAgentK — APPROVED** [review]
LGTM

**Comment by TheRealAgentK** [comment]
Ok, so this has a bunch of conflicts, leaving to @miquelbeltran to have a look at resolving before merge.

File: examples/express-sample/app.js [hunk 1]
```diff
@@ -23,8 +24,7 @@ app.set("view engine", "ejs");
 // Add the Raygun breadcrumb Express handler
 app.use(raygunClient.expressHandlerBreadcrumbs);
 
-// uncomment after placing your favicon in /public
-// app.use(favicon(__dirname + '/public/favicon.ico'));
+app.use(favicon(__dirname + "/public/favicon.ico"));
```
Comment (sumitramanga) [inline examples/express-sample/app.js:27]: Will it throw an error if no favicon is present?

File: lib/types.ts [hunk 2]
```diff
@@ -196,6 +196,9 @@ export function callVariadicCallback<T>(
 
 export type Callback<T> = CallbackNoError<T> | CallbackWithError<T>;
 
+/**
+ * Internal type, sent to the Raygun.com as part of MessageDetails
```
Comment (sumitramanga) [inline lib/types.ts:200]:
```suggestion
 * Internal type, sent to the Raygun API as part of MessageDetails
```

Does this make sense to change?

**Review by sumitramanga — CHANGES_REQUESTED** [review]
Just one thing to consider. Thank you for implementing my feedback 😊

File: lib/types.ts (hunk 2 above)
Comment (TheRealAgentK) [reply lib/types.ts:200]: We could then even say `Raygun Crash Reporting API`

File: examples/express-sample/app.js (hunk 1 above)
Comment (miquelbeltran) [reply examples/express-sample/app.js:27]: A favicon has been included in the repo

**Review by sumitramanga — APPROVED** [review]
Thank you for the quick turn around! :shipit:

**Pull request merged**